walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -113,13 +119,12 @@ and available plugins that will be used for source code generation
 `yarn graphback generate`
 . Review `./server/src/schema/schema.qraphql`.
 This file contains your model along with access methods that were enabled in configuration.
-. Review generated resolver files in `./server/src/resolvers/resolvers.ts`
 This file contains methods used to fetch data. Each individual method will use
```

Comment:
Remove this sentence.

Reply:
removed in 95370f7376c73c14c90b1975352d1e38b20507e2